src/interfaces/EditInterface/EditInterface.js

```diff
@@ -110,6 +113,7 @@ class EditInterface extends Component {
                 <Field
                   label={<FormattedMessage id="ui-organizations.interface.name" />}
                   name="name"
+                  validate={[Required]}
```

Comment:
```suggestion
                  validate={validateRequired}
```

Reply:
I haven't seen the similar syntax regarding required input fields and made like how it looks in this module. Thanks, i'll fix it